README.md

```diff
@@ -34,8 +37,8 @@ Use this when you need to repeatedly allocate and free a bunch of temporary buff
 ### Disadvantages over GC:
 
 * Can leak memory contents. Unlike the go GC, we *don't* zero memory.
-* All buffers have a capacity of a power of 2. This is fine if you either (a) actually need buffers with this size or (b) expect these buffers to be temporary.
-* Requires that buffers be returned explicitly. This can lead to race conditions and memory corruption if the buffer is released while it's still in use.
+* All buffers have a capacity of a power of 2. This is fine if you either expect these buffers to be temporary.
```

Comment:
Think you accidentally a clause here 😉 (Why is powers-of-two meaningful?)

Reply:
> Why is powers-of-two meaningful?

It's a convenient grouping that means you'll only ever over-allocate by 2x.